docs/dep/dep-0001.rst

```diff
@@ -0,0 +1,599 @@
+document providing information to the DebOps community, or describing
+a new feature for DebOps or its processes or environment. The DEP
+should provide a concise technical specification of the feature and a
+rationale for the feature.
+
+We intend DEPs to be the primary mechanisms for proposing major new
+features, for collecting community input on an issue, and for
+documenting the design decisions that have gone into DebOps. The DEP
+author is responsible for building consensus within the community and
+documenting dissenting opinions.
+
+Because the DEPs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+feature proposal [1]_.
+
+DEPs are modeled after similar concept popularized by the
+`Python Enhancement Proposals`__ (PEPs).
+
+.. __: https://www.python.org/dev/peps/pep-0001/
+
+
+DEP Types
+---------
+
+There are three kinds of DEP:
+
+1. A **Standards Track** DEP describes a new feature or implementation for
+   DebOps. It may also describe an interoperability standard that will be
+   supported outside the standard DebOps environment for current DebOps
+   versions before a subsequent DEP adds standard environment support in
+   a future version.
+
+2. An **Informational** DEP describes a DebOps design issue, or
+   provides general guidelines or information to the DebOps community,
+   but does not propose a new feature. Informational DEPs do not
+   necessarily represent a DebOps community consensus or
+   recommendation, so users and implementers are free to ignore
+   Informational DEPs or follow their advice.
+
+3. A **Process** DEP describes a process surrounding DebOps, or
+   proposes a change to (or an event in) a process.  Process DEPs are
+   like Standards Track DEPs but apply to areas other than the DebOps
+   source code itself. They may propose an implementation, but not to
+   DebOps's codebase; they often require community consensus; unlike
+   Informational DEPs, they are more than recommendations, and users
+   are typically not free to ignore them.  Examples include
+   procedures, guidelines, changes to the decision-making process, and
+   changes to the tools or environment used in DebOps development.
+   Any meta-DEP is also considered a Process DEP.
+
+
+DEP Workflow
+------------
+
+
+DebOps maintainers
+~~~~~~~~~~~~~~~~~~
+
+There are several references in the DEP to the "DebOps maintainers". Original
+creator and maintainer of DebOps is Maciej Delmanowski. There are a few other
+people currently active in DebOps development, you can see the list of
+contributors at: https://github.com/debops/debops/graphs/contributors.
+
+
+DEP Editors
+~~~~~~~~~~~
+
+The DEP editors are individuals responsible for managing the administrative
+and editorial aspects of the DEP workflow (e.g. assigning DEP numbers and
+changing their status).  See `DEP Editor Responsibilities & Workflow`_ for
+details. The current editors are:
+
+- Maciej Delmanowski
```

Comment:
I might add myself when I have more time, but not right now, at least not officially. But I will try to keep up and should meet the DEP Editor requirements. But maybe don’t request a review from me where you think I am not involved in the discussion or which is unrelated to me, for now.

Reply:
I'm aware of this and I did not intentionally add you or anybody else just yet, community is still small.

DEPs (and PEPs which they were based on) are designed for long-term discussions and mesh perfectly well with `git` history. This allows for interested parties to voice their concens at their own pace. But since this PR only introduces them, I wanted to have a few other people in the community review them beforehand to polish all the issues we might have.